app/src/visionglass/cpp/DeviceDelegateVisionGlass.cpp

```diff
@@ -314,7 +328,6 @@ DeviceDelegateVisionGlass::Resume() {
 
 void
 DeviceDelegateVisionGlass::RecenterView() {
-  m.headingMatrix = vrb::Matrix::Identity();
```

Comment:
Any reason why we want this empty private method now ?

Reply:
Not really we can remove it